src/ui/public/timepicker/kbn_global_timepicker.js

```diff
@@ -25,6 +26,73 @@ UiModules
       $rootScope.toggleRefresh = () => {
         timefilter.refreshInterval.pause = !timefilter.refreshInterval.pause;
       };
+
+      // functions for the time navigation buttons
```

Comment:
minor: consider removing this comment, I think the code is nice & dense enough to be self explanatory.